frontend/readhistory.lua

```diff
@@ -301,9 +301,9 @@ end
 
 --- Reloads history from history_file.
 -- @treturn boolean true if history_file has been updated and reload happened.
-function ReadHistory:reload()
+function ReadHistory:reload(force_read)
     assert(self ~= nil)
-    if self:_read() then
+    if self:_read(force_read) then
```

Comment:
This seems a bit roundabout?

Reply:
Compared to something like ` self:_read() or force_read` I mean.

Reply:
Do you mean the name of the argument? `no_time_check`?

Reply:
Never mind, the diff here on GH hid the pertinent part of _read(), it looked like it was just returning true or false.